<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QProcessInfoWidget</class>
 <widget class="QWidget" name="QProcessInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Process Browser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QCheckBox" name="systemProcessesCheckBox">
     <property name="toolTip">
      <string>Show system processes or hide them.</string>
     </property>
     <property name="text">
      <string>System Processes</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="refreshToolButton">
     <property name="toolTip">
      <string>Refresh the list of processes.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="view-refresh">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeWidget" name="processTreeWidget">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>PID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Program  Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Command Line</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="programNameLineEdit">
     <property name="toolTip">
      <string>Search program name in the list of processes. &quot;*&quot; is allowed.</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Search Program Name...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="userNameLineEdit">
     <property name="toolTip">
      <string>Search user name in the list of processes. &quot;*&quot; is allowed.</string>
     </property>
     <property name="placeholderText">
      <string>Search User Name...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
